README.md

```diff
@@ -20,13 +69,73 @@ Third, install the [Bower npm resolver](https://github.com/mjeanroy/bower-npm-re
+
+#### Creating a Bluemix Cloudant NoSQL DB service
+
+Sign up for an [IBM Bluemix](https://console.ng.bluemix.net/) account, if you do not already have one.
+
+Once you are logged in to Bluemix, create a new Cloudant instance on the [Cloudant NoSQL DB Bluemix Catalog](https://console.ng.bluemix.net/catalog/services/cloudant-nosql-db) page. This should take you to a page representing the newly-created service instance. Click the "Service credentials" link. You should have one set of service credentials listed. Click "View credentials" which should show you a JSON object containing your service credentials. Copy the value for the `url` key to your clipboard (the value will be in the form of `https://username:password@uniqueid-bluemix.cloudant.com`).
+
+## Database and replication setup
+1. [Create the remote database](#1-create-the-remote-database)
+1. [Enable CORS](#2-enable-cors)
+1. [Set the replication target](#3-set-the-replication-target)
+
+### 1. Create the remote database
+
+Use your Cloudant or CouchDB dashboard to create a database. Select the Databases icon on the left and then use the `Create Database` button to create the "shopping-list" database.
+The Shopping List app can be used locally before the database exists, but cannot sync
+until the remote database is completed.
+
+![](doc/source/images/create_db.png)
+
+### 2. Enable CORS
+
+Cross-Origin Resource Sharing (CORS) needs to be enabled. Use your Cloudant or CouchDB dashboard to enable it. The CORS options are under the account settings or config depending on your version. Enable CORS and restrict the domain as needed for security.
+
+![](doc/source/images/enable_cors.png)
+
+### 3. Set the replication target
+
+Run the Shopping List app and use the `Replicator` form to enter your Database URL.
+If you use the Bluemix Cloudant URL taken from the service credentials as described above, the URL includes user and password GUIDs.
+
+Add `/shopping-list` to the URL to connect to the database that you created.
+
+![](doc/source/images/replicator.png)
+
+# Using the app
+
+The app allows you to create a shopping list by clicking on the plus sign. Click on the list to see its items. Then, you can add items to the list by clicking the plus sign. There is a checkbox to allow you to mark the items complete as you buy load up your cart.
+
+When you have not configured your Replication Target or when you are offline, the lists will not sync. One good way to test this is to run two browsers. You can use Chrome and Firefox and have different lists in each.
+
+When you go online and have the database and CORS enabled and the Replication Target is set, the shopping lists will sync. You will then be able to use both lists from either browser.
+
+![](doc/source/images/shopping_lists.png)
+
+## Epic features
```

Comment:
Perhaps "Features as epics" might make more sense as the heading here (or just "Features")?